connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -826,9 +826,6 @@ message MapPartitions {
 
   // (Required) Input user-defined function.
   CommonInlineUserDefinedFunction func = 2;
-
-  // (Optional) Whether to use barrier mode execution or not.
-  optional bool is_barrier = 3;
```

Comment:
This is a breaking change right?

Reply:
yes, but this field was added in master only. So it is not a breaking change against 3.4